### PR TITLE
feat(console): replace wget with curl, user login prompt, warn dialog

### PR DESCRIPTION
**Pull request by peintnermax**

### CODE REVIEW

File: console/src/app/modules/accounts-card/accounts-card.component.ts [hunk 1]
```diff
@@ -45,21 +45,27 @@ export class AccountsCardComponent implements OnInit {
         }
     }
 
-    public selectAccount(loginHint?: string, idToken?: string): void {
+    public selectAccount(loginName?: string): void {
         const configWithPrompt: Partial<AuthConfig> = {
             customQueryParams: {
-                prompt: 'select_account',
+                // prompt: 'select_account',
```
Comment (fforootd) [inline console/src/app/modules/accounts-card/accounts-card.component.ts:51]: do we want to remove the prompt param?

File: console/src/assets/i18n/de.json [hunk 2]
```diff
@@ -230,7 +230,11 @@
         },
         "DOMAINS": {
             "TITLE":"Domains",
-            "DESCRIPTION":"Konfigurieren Sie Ihre Domains mit denen sich Ihre Nutzer einloggen können."
+            "DESCRIPTION":"Konfigurieren Sie Ihre Domains mit denen sich Ihre Nutzer einloggen können.",
+            "DELETE": {
+                "TITLE":"Domain löschen?",
+                "DESCRIPTION":"Sie sind im Begriff eine Domain aus Ihrer Organisation zu löschen. Beachten Sie, dass dies unwiederrufbares Verhalten verursachen kann."
+            }
```
Comment (fforootd) [inline console/src/assets/i18n/de.json:237]:
```suggestion
            "DESCRIPTION":"Konfigurieren Sie die Domains mit denen sich Ihre Nutzer einloggen können.",
            "DELETE": {
                "TITLE":"Domain löschen?",
                "DESCRIPTION":"Sie sind im Begriff eine Domain aus Ihrer Organisation zu löschen. Ihre Benutzer können diese nach dem Löschen nicht mehr für den Login nutzen."
            }
```

File: console/src/assets/i18n/en.json [hunk 3]
```diff
@@ -231,7 +231,11 @@
         },
         "DOMAINS": {
             "TITLE":"Domains",
-            "DESCRIPTION":"Configure your domains on which your users will be registered."
+            "DESCRIPTION":"Configure your domains on which your users will be registered.",
+            "DELETE": {
+                "TITLE":"Delete domain?",
+                "DESCRIPTION":"You are about to delete one of your domains. Note that this action can trigger an irrevocable behaviour."
+            }
```
Comment (fforootd) [inline console/src/assets/i18n/en.json:238]:
```suggestion
            "DESCRIPTION":"Configure your domains on which your users will be registered.",
            "DELETE": {
                "TITLE":"Delete domain?",
                "DESCRIPTION":"You are about to delete one of your domains. Note that your users can no longer use this domain for their logins."
            }
```

File: console/src/app/modules/accounts-card/accounts-card.component.ts (hunk 1 above)
Comment (peintnermax) [reply console/src/app/modules/accounts-card/accounts-card.component.ts:51]: yes, oidc standard states you can always send a login hint

**Pull request merged**